client/pokemon/pokemon.ctrl.js

```diff
@@ -0,0 +1,58 @@
+/**
+ *
+ */
+module.exports = function($routeParams, $http, $scope) {
+  const self = this;
+  self.pokemon = self.pokemon || {};
+  self.name = self.pokemon.speciesName;
+  self.nickname = self.pokemon.nickname;
+  self.user = self.pokemon.user;
+  self.id = $routeParams.pokemonid || self.pokemon.id;
+  self.dexNo = self.pokemon.dexNo;
+  self.ball;
+  self.item;
+  self.level = self.pokemon.level;
+  self.nature = self.pokemon.natureName;
+  self.ability = self.pokemon.abilityName;
+  self.ot = self.pokemon.ot;
+  self.tid = self.pokemon.tid;
+  self.ivs = self.pokemon.ivHp + '/' +
+              self.pokemon.ivAtk + '/' +
+              self.pokemon.ivDef + '/' +
+              self.pokemon.ivSpe + '/' +
+              self.pokemon.ivSpAtk + '/' +
+              self.pokemon.ivSpDef;
+
+  self.language = self.pokemon.language;
+  // self.ballNameUrl = self.ballName.replace(' ', '-').toLowerCase();
+  self.heldItemUrl = self.pokemon.heldItemName.replace(' ', '-').toLowerCase();
```

Comment:
This will throw an error if the Pokémon isn't holding an item, because `pokemon.heldItemName` will be `null`.